server.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var http = require('http');
+var url = require('url');
+// Found a sweet time format on stack overflow by L0j1k
+var currentTime = new Date();
+/* jshint ignore:start */
+var dateTime = "Server time: " + (currentTime.getMonth()+1) + "/"
+                + currentTime.getDate() + "/" 
+                + currentTime.getFullYear() + " @ "  
+                + currentTime.getHours() + ":"  
+                + currentTime.getMinutes() + ":" 
+                + currentTime.getSeconds();
+/* jshint ignore:end */
+
+var server = http.createServer(function(req, res) {
+
+	if(req.url === '/time') {
+		res.writeHead(200, {'Content-Type': 'application/json'});
```

Comment:
You want to make a new date inside this if statement. The date that is sent back is the date that the server has been built on.